src/toolkits/object_detection/one_shot_object_detection/util/parameter_sampler.cpp

```diff
@@ -22,6 +22,14 @@ ParameterSampler::ParameterSampler(size_t width, size_t height, size_t dx,
 
 double deg_to_rad(double angle) { return angle * M_PI / 180.0; }
 
+std::vector<double> vector_deg_to_rad(std::vector<double> means_in_degrees) {
+  std::vector<double> means_in_radians;
```

Comment:
It's also good practice to add a reserve() call when using the push_back pattern and you know how many elements will be added.  This avoids a lot of unnecessary reallocations and copies.  Here, add 
`means_in_radians.reserve(means_in_degrees.size());`

Reply:
Done.